test/test_legacy_event_converter.rb

```diff
@@ -62,7 +62,7 @@ def load_legacy_event filename, url_path: nil
   end
 
   it "converts pubsub_utf8.json" do
-    event = load_legacy_event "pubsub_utf8.json"
+    event = load_legacy_event "pubsub_utf8.json", encoding: "ASCII-8BIT"
```

Comment:
The `external_encoding` of `read_input_json` `input` is `ASCII-8BIT`, so I set it.
When I set `ASCII-8BIT`, and let using `encoding` alone, I got the same error as I got on the real CloudFunction.